### PR TITLE
Node 14 Support (Experimental)

### PR DESCRIPTION
**Pull request by acburdine**

### CODE REVIEW

File: test/unit/tasks/yarn-install-spec.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ const modulePath = '../../../lib/tasks/yarn-install';
 const errors = require('../../../lib/errors');
 
 describe('Unit: Tasks > yarn-install', function () {
+    beforeEach(() => {
+        process.env = {};
```
Comment (vikaspotluri123) [inline test/unit/tasks/yarn-install-spec.js:16]: Do we have any tests that depend on `NODE_ENV=test` (or something like that)

Comment (acburdine) [reply test/unit/tasks/yarn-install-spec.js:16]: Not that I'm aware of - the only NODE_ENV usage I recall seeing was when we set `NODE_ENV = 'production'` when calling `yarn install `, and in that case it's a hardcoded value.

File: lib/utils/local-process.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ Ensure the content folder has correct permissions and try again.`));
             });
 
             // Stick the pid into the pidfile so we can stop the process later
-            fs.writeFileSync(path.join(cwd, PID_FILE), cp.pid);
+            fs.writeFileSync(path.join(cwd, PID_FILE), `${cp.pid}`);
```
Comment (vikaspotluri123) [inline lib/utils/local-process.js:47]: Do you know what broke here?

**Review by vikaspotluri123 — APPROVED** [review]
LGTM 👍🏾

File: lib/utils/local-process.js (hunk 2 above)
Comment (acburdine) [reply lib/utils/local-process.js:47]: Yeah so from what I was seeing Node 14 tightened up the type checking for the writeFile methods - it now errors if you pass it contents that aren't either a string or some sort of data buffer

Comment (vikaspotluri123) [reply lib/utils/local-process.js:47]: Aah I see. Could we do this to make it more obvious?

```suggestion
            fs.writeFileSync(path.join(cwd, PID_FILE), cp.pid.toString());
```

Comment (acburdine) [reply lib/utils/local-process.js:47]: Yeah the `toString` call makes sense - I'll make that change in the morning before I merge 🙂

**Pull request merged**